vtex/address-form
# CHK-815: Add the county capital cities and their corresponding postal codes

#### What is the purpose of this pull request?

Add the county capital cities and their corresponding postal codes; additional, some names were capitalized. For Giurgiu and Ilfov counties, the postal codes were missing a "0" before the actual code, so I added them.

#### What problem is this solving?

When trying to select a city that is a county capital, the city was not actually in the list. Also, for the Giurgiu and Ilfov counties, it was impossible to proceed with choosing one city, as their postal codes were incomplete(missing a 0 in front of the actual number).

#### How should this be manually tested?

Try adding a new address, and for the chosen city, we should have the correct postal code, instead of the general one used before the update.

#### Screenshots or example usage

#### Types of changes
